src/lib/core/crash.h

```diff
@@ -9,23 +9,124 @@
 extern "C" {
 #endif /* defined(__cplusplus) */
 
-/**
- * PATH_MAX is too big and 2K is recommended
- * limit for web address.
- */
-#define CRASH_FEEDBACK_HOST_MAX 2048
+#if defined(TARGET_OS_LINUX) && defined(__x86_64__)
+# define HAS_GREG
+#endif
 
-/**
- * Initialize crash subsystem.
+#ifdef HAS_GREG
+/*
+ * Values of x86 registers.
  */
-void
-crash_init(const char *tarantool_bin);
+struct crash_greg {
+	/* r8 register value */
+	uint64_t r8;
```

Comment:
Nit: Please include `stdint.h` and other headers needed to compile this one.

Reply:
Fixed.
```
+#include <stdint.h>
+#include <time.h>
```